File: timesTable/app/src/main/java/com/example/timestable/MainActivity.java
package com.example.timestable;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import android.widget.SeekBar;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    private ListView listView;

    public void generateTimesTable(int number){
        List<String> resultArr = new ArrayList<>();
        int i =1;
        while(i<=10){
            resultArr.add(""+(i)*number);
            i++;
        }
//
        ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(this,android.R.layout.simple_list_item_1,resultArr);
        arrayAdapter.addAll(resultArr);
        listView.setAdapter(arrayAdapter);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        listView = findViewById(R.id.listView);

        SeekBar number = findViewById(R.id.seekBar);
        number.setMax(20);
        number.setProgress(10);
        generateTimesTable(10);


//        final ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(this,android.R.layout.simple_list_item_1,resultArr);

        number.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
                int currentNumber = progress;
                seekBar.setProgress(progress);
                Log.i("Info", String.valueOf(currentNumber));

                generateTimesTable(currentNumber);


            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {

            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {

            }
        });
    }
}

File: CurrencyConvertor/app/src/main/java/com/example/currencyconvertor/MainActivity.kt
package com.example.currencyconvertor

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun convertCurrency(view: View){
        var currency = findViewById<EditText>(R.id.currencyText);
        var convert = currency.text.toString().toDouble() * 76.14;

        Toast.makeText(this,convert.toString(),Toast.LENGTH_SHORT).show();
    }
}
